src/App.spec.jsx

```diff
@@ -4,6 +4,7 @@ import { App } from './App';
 
 describe('App', () => {
   beforeEach(() => {
+    // eslint-disable-next-line react/jsx-filename-extension
```

Comment:
do you need this?

Reply:
do not how to fix another way. Prooblem is JSX not allowed in files with extension '.jsx'.